<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Page de connexion</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="resetButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>RÃ©initialiser</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginTB">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Login : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordTB">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="connectButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Connexion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot de passe : </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Liste des cours inscrits</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewCoursInscrits">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>551</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>loginTB</tabstop>
  <tabstop>passwordTB</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>tableViewCoursInscrits</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
